### DISCUSSION TITLE
Now that query is a self contained package, is it possible to use it with Node?

### DISCUSSION THREAD

**Discussion by anthonyma94** [TanStack/query#4120] [Ideas]
Caching in server-side is just as important as client-side, but I have yet to see a good caching library for Node. React query has been extremely helpful in dealing with API calls, I can see this functionality being used in an Express server for example to reduce the number of calls to DBs and such.

Is there a plan to create an adapter for Node in the future?

**Comment by TkDodo** [comment]
Not sure what an adapter would need to do? I guess using `@tanstack/query-core` should work fine in node...
